src/main/java/taxi/model/Car.java

```diff
@@ -46,4 +46,19 @@ public Manufacturer getManufacturer() {
     public List<Driver> getDrivers() {
         return drivers;
     }
+
+    @Override
+    public String toString() {
+        return "Car{"
+                + "id="
+                + id
+                + ", model='"
+                + model
+                + '\''
+                + ", manufacturer="
+                + manufacturer
+                + ", drivers="
+                + drivers
+                + '}';
```

Comment:
```suggestion
        return "Car {"
                + "id=" + id
                + ", model='" + model
                + ", manufacturer=" + manufacturer
                + ", drivers=" + drivers + '}';
```
let's make this more readable